library/alloc/src/string.rs

```diff
@@ -1042,7 +1044,7 @@ impl String {
     #[must_use]
     #[stable(feature = "string_as_str", since = "1.7.0")]
     pub fn as_mut_str(&mut self) -> &mut str {
```

Comment:
FWIW it should be fine to also do this with the `mut` methods

Reply:
I went ahead and added these